src/SprykerEco/Zed/Unzer/Business/UnzerFacadeInterface.php

```diff
@@ -403,4 +404,22 @@ public function expandQuoteWithUnzerCredentials(QuoteTransfer $quoteTransfer): Q
      * @return \Generated\Shared\Transfer\UnzerPaymentTransfer|null
      */
     public function findUpdatedUnzerPaymentForOrder(OrderTransfer $orderTransfer): ?UnzerPaymentTransfer;
+
+    /**
+     * Specification:
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.quote` transfer property to be set.
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.paymentMethodKey` transfer property to be set.
+     * - Gets main marketplace Unzer credentials from persistence.
+     * - Performs Unzer get payment methods API call.
+     * - Returns payment method related Unzer credentials.
```

Comment:
Exception that can be thrown in Resolver should be mentioned here

Reply:
Fixed.